vs/server/src/setup.ts

```diff
@@ -54,6 +54,11 @@ export async function generateBuildFile(workspaceUri: string, type: string) {
 				fs.writeFileSync(path.join(cwd, `makefile`), makeProj.getMakefile().join(`\n`));
 				break;
 
+			case `imd`:
+				const impactMarkdown = new ImpactMarkdown(cwd, targets, []);
+				fs.writeFileSync(path.join(cwd, `impact.md`), impactMarkdown.getContent().join(`\n`));
+				break;
+
```

Comment:
ah so 'imd' stands for 'Impact Markdown' and this is a file generated to represent impact of ????

Reply:
This is for an impot report of the entire project.

Refer to https://github.com/IBM/sourceorbit/pull/84#discussion_r1788222431